cms/envs/bok_choy.env.json

```diff
@@ -75,7 +75,9 @@
         "PREVIEW_LMS_BASE": "preview.localhost:8003",
         "ENABLE_CONTENT_LIBRARIES": true,
         "ENABLE_SPECIAL_EXAMS": true,
-        "SHOW_LANGUAGE_SELECTOR": true,
+        "LANGUAGE_SELECTOR": {
```

Comment:
Please do not add dictionaries to the feature flag list, feature flags should be flags (boolean settings).